index.html

```diff
@@ -875,6 +875,47 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
           Location Discovery</a>.
         </p>
       </section>
+      <section id="append-mode">
+        <h2>Append Mode</h2>
+        <p>
+          The [[!SOLIDWEBAC]] <a href="https://github.com/solid/web-access-control-spec#modes-of-access">Modes of
+          Access</a> section mentions the <code>acl:mode</code> value <code>acl:Append</code>.  In the context of a
+          Fedora implementation, <code>acl:Append</code> should be understood as operations that only append, such as
+          <code>POST</code>ing to a container, or performing a <code>PATCH</code> that only adds triples.
+        </p>
+        <section id="appendLDPRS">
+          <h3>LDP-RS</h3>
+          <p>
+            When a client is allowed to perform <code>acl:Append</code> operations on an <a>LDP-RS</a>:
+          </p>
+          <ul>
+            <li>A <code>DELETE</code> request MUST be denied</li>
+            <li>A <code>PATCH</code> request that deletes triples MUST be denied</li>
+            <li>A <code>PATCH</code> request that only adds triples SHOULD be allowed</li>
+            <li>A <code>PUT</code> request on an existing resource MUST be denied</li>
```

Comment:
Maybe this should be a SHOULD?  You could PUT the original resource plus some new triples.

Reply:
I agree that the result would appear to be the appending of triples, but the semantics of `PUT` is to remove the existing triples and replace them with the provided triples... which is different than "append".

I would prefer we stick with the existing `MUST` language.

Reply:
I'm fine with that.